client/blocks/site/index.jsx

```diff
@@ -98,67 +85,16 @@ class Site extends Component {
 		this.props.onMouseLeave( event, this.props.site.ID );
 	};
 
-	onViewSiteClick = ( event ) => {
-		const { isPreviewable, siteSlug } = this.props;
```

Comment:
Both of these can be removed from the `mapStateToProps` HoC, since they're used.

https://github.com/Automattic/wp-calypso/blob/ee3bf9ffe8c76dcabcaff40f2d7095a5d026cc5a/client/blocks/site/index.jsx#L234-L235